src/manifest/adaptation.ts

```diff
@@ -110,13 +110,6 @@ export default class Adaptation {
    */
   public manuallyAdded? : boolean;
 
-  /**
-   * `false` if from all Representation from this Adaptation, none is decipherable.
-   * `true` if at least one is known to be decipherable.
-   * `undefined` if this is not known for at least a single Representation.
-   */
-  public decipherable? : boolean;
-
```

Comment:
Removed that one as it was never used (only the `decipherable` property from a `Representation` is used) and I found that it complexified the already complex handling of "decipherability" in the player's code (in that it is handled both in the `StreamOrchestrator` for cleaning the buffer and asking to flush it, and in the `AdaptationStream` to re-construct the list of `Representation` we have to choose from).